faq.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: 'Frequently Asked Questions'
+---
+
+A set of frequently asked questions and useful links about the **DITA Open Toolkit**.
+{: .lead}
+
+## What is DITA?
+
+**DITA** is the Darwin Information Typing Architecture, an **XML standard** for authoring modular documents with an emphasis on reuse, maintained by the [DITA Technical Committee at OASIS][1].
+
+## What is the DITA Open Toolkit?
+
+The **DITA Open Toolkit** (DITA-OT) is a publishing tool used to convert **DITA** content into various output formats.
+
+## How do I use the DITA Open Toolkit?
+
+You can [download][2], [install][3] and [build output][4] from **DITA** content using the command line.
+
+Besides this, there are applications which come with the **DITA Open Toolkit** bundled. For example Oxygen XML Editor usually comes bundled with the latest **DITA Open Toolkit**. Oxygen provides visual means to run the bundled **DITA Open Toolkit** using a concept called [transformation scenarios][5].
+
+## What version of DITA Open Toolkit should I use?
+
+You should try to use the latest **DITA Open Toolkit** release available on the official [download][6] page.
```

Comment:
This page will be on the dita-ot.org page. IMO remove "official".

Reply:
Resolved in 6bf4448.